drivers/include/periph/adc.h

```diff
@@ -30,6 +30,18 @@
  * waiting for the result of a conversion (e.g. through putting the calling
  * thread to sleep while waiting for the conversion results).
  *
+ * @section     sec_pm (Low-) power implications
```

Comment:
you mean that I cannot have `sec_pm` in the adc and than again `sec_pm` in say the spi header? The solution being to prefix the section name with e.g. `sec_periph_adc_pm`?!